src/TuringTutorials.jl

```diff
@@ -1,186 +1,134 @@
 module TuringTutorials
 
-import IOCapture
+import Requires
```

Comment:
```suggestion
using Requires: @require
```
Makes it more clear that we don't extend anything from `Requires`.

Reply:
`import Requires` is not sufficient to extend any method or macro from Requires directly. It only loads `Requires` in the namespace and is equivalent with `using Requires: Requires`.

Reply:
> `import Requires` is not sufficient to extend any method or macro from Requires directly.

No, but it makes it possible and that's why many people stick to explicit loading of symbols via `using X: y`.

Reply:
No, it doesn't. You still have to write `Requires.f(...) = ...` if you want to extend `Requires.f`.

It would be different if you use `import Requires: f`. Then you could extend it with `f(...) = ...`.

Reply:
Irregardless of what's best here, this shouldn't stand in the way of merging.